cmd/notary/cert.go

```diff
@@ -63,29 +64,30 @@ func certRemove(cmd *cobra.Command, args []string) {
 	}
 
 	var certsToRemove []*x509.Certificate
+	var certFoundByID *x509.Certificate
 
 	// If there is no GUN, we expect a cert ID
 	if certRemoveGUN == "" {
 		certID := args[0]
 		// This is an invalid ID
-		if len(certID) != idSize {
+		if len(certID) != notary.IDSize {
```

Comment:
My reasoning was that a message saying that the ID is invalid is better than people receiving a "ID does not exist" because they forgot 1 char, or something of the sort.